utils/ProgressBar.go

```diff
@@ -208,15 +203,23 @@ func (pb *ProgressBar) String(termWidth int) string {
 	rightEdge := builder.String()
 
 	if rightColumnWidth > len(rightEdge) {
-		rightEdge = strings.Repeat(" ", rightColumnWidth-len(rightEdge)) + rightEdge
+		if pb.label == "🕸  Building DAG" {
```

Comment:
Is this still needed after #1 ?

Reply:
Weirdly yes.

Reply:
This feels super hacky, I don't like the idea of including the label text hard coded twice in the progress bar, given it's so far agnostic to what it's operating on.

Reply:
Agreed! Otherwise I'm not sure where the problem is arising from. I was focusing on the if statements for conditional treatment of this case. But nothing seemed obvious to what was generating the issue.